lua/dapui/init.lua

```diff
@@ -186,9 +186,11 @@ function dapui.eval(expr, args)
     prev_expr = expr
     local elem = dapui.elements.hover
     elem.set_expression(expr, args.context)
-    local line_no = async.fn.screenrow()
-    local col_no = async.fn.screencol()
```

Comment:
`screenrow()` and `screencol()` always seem to output `1`.

See here: https://github.com/neovim/neovim/issues/15754#issuecomment-924905304